docs/schemas/ProductRegistrationEvidenceDocument.json

```diff
@@ -0,0 +1,249 @@
+{
+	"$id": "https://w3id.org/traceability/schemas/ProductRegistrationEvidenceDocument.json",
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"$comment": "{\"term\": \"ProductRegistrationEvidenceDocument\", \"@id\": \"https://w3id.org/traceability/ProductRegistrationEvidenceDocument\"}",
```

Comment:
propbably this URL should look like https://w3id.org/traceability#ProductRegistrationEvidenceDocument

and I would expect this term to be defined in the spec in short order.... please open an issue to track making this URL resolvable in order for this PR to be accepted.